xcode/Subconscious/Shared/Services/DatabaseService.swift

```diff
@@ -451,7 +476,7 @@ final class DatabaseService {
 
         let results = try database.execute(
             sql: """
-            SELECT slashlink, modified, excerpt
+            SELECT slashlink, modified, length(body) AS length, excerpt
```

Comment:
Noting that the excerpt field is created during write by way of `Memo.excerpt()` for `MemoRecord()`.

We truncate this field so that we can reduce the amount of text that has to be returned for list queries. We'll want to make sure that `Memo.excerpt()` uses the same truncation length/logic as `EntryStub`'s excerpt generation code.

I believe this is the case.